migrations/005_Scores.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TYPE score_status AS ENUM (
+    'failed',
+    'submitted',
+    'best',
+    'mods'
+);
+
+CREATE TABLE scores (
+    id bigserial NOT NULL PRIMARY KEY,
+    beatmap_id int NOT NULL REFERENCES beatmaps (id),
+    user_id int NOT NULL REFERENCES users (id),
+    checksum character(32) NOT NULL,
+    status score_status NOT NULL,
```

Comment:
Should we make `status` an enum, or leave it as a normal integer, like we did with beatmap submission?

Reply:
I'd say consistency is key, even though enums are nice